src/FactorGraph.jl

```diff
@@ -880,7 +880,7 @@ function initManual!(dfg::AbstractDFG, sym::Symbol, pX::BallTreeDensity)::Nothin
 end
 function initManual!(dfg::AbstractDFG, sym::Symbol, usefcts::Vector{Symbol})::Nothing
   @info "initManual! $sym"
-  pts = predictbelief(dfg, sym, usefcts)
+  pts = predictbelief(dfg, sym, usefcts)[1]
```

Comment:
This looks like a bug, found with MultiHypo3Door example

Reply:
oh thats weird, I had been using that quite a bit at some point... Ah think i see what happened:
```julia
return pGM, sum(inferdim)
```

The newer code needs both the `points` and the `inferdim`.  Just make sure that the use cases inside and below `initManual` do not also require `inferdim`?  Perhaps the name `pts` is misleading in the newer version

Reply:
Doesn't look like it, its only `Xpre = AMP.manikde!(pts, getSofttype(vert) |> getManifolds )`